pkg/scheduler/internal/cache/debugger/dumper.go

```diff
@@ -44,9 +44,9 @@ func (d *CacheDumper) DumpAll() {
 // dumpNodes writes NodeInfo to the scheduler logs.
 func (d *CacheDumper) dumpNodes() {
 	dump := d.cache.Dump()
-	klog.Info("Dump of cached NodeInfo")
+	klog.InfoS("Dump of cached NodeInfo")
 	for name, nodeInfo := range dump.Nodes {
-		klog.Info(d.printNodeInfo(name, nodeInfo))
+		klog.InfoS(d.printNodeInfo(name, nodeInfo))
```

Comment:
Passing a message string with line breaks to `InfoS` has the same problem as passing it a value: it gets printed as quoted string on a single line.

You have to keep using `klog.Info` here.

Reply:
Okay, done!